src/test/groovy/org/ods/doc/gen/leva/doc/fixture/LevaDocDataFixture.groovy

```diff
@@ -26,6 +26,7 @@ class LevaDocDataFixture {
         data.build = buildJobParams(projectFixture)
         data.git =  buildGitData(projectFixture)
         data.openshift = [targetApiUrl:"https://openshift-sample"]
+        data.repositories = new ProjectRepositoryFixture().load()
```

Comment:
1- The data in getModuleDatais the same you put here.
2- Shared is sending this data for everytype of doc?

Reply:
1-That is not true. While the previous contains only info for a component, new code cotains different information for each code.
2-If you are asking this, you did not expend enough time reading my code.